staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -157,12 +158,14 @@ func (rl *respLogger) Addf(format string, data ...interface{}) {
 
 func (rl *respLogger) LogArgs() []interface{} {
 	latency := time.Since(rl.startTime)
+	auditID := request.GetAuditID(rl.req)
```

Comment:
it cannot be empty, right ?

Reply:
if `WithAuditID` is wired, it cannot be empty.